gcc/rust/util/rust-mapping-common.h

```diff
@@ -32,16 +32,31 @@ typedef uint32_t HirId;
 // refers to any top-level decl in HIR
 typedef uint32_t LocalDefId;
 // refers to <Crate><DefId>
```

Comment:
Is this comment still accurate?

Reply:
Thanks. Removed line 34 comment. I am assuming line 32 comment is good because I did not make any changes to that part.